src/Repository.java

```diff
@@ -83,5 +85,59 @@ public static Repository getInstance() {
         }
         return result;
     }
+    
+    public void getSongs() throws SQLException {
+    	statement = connection.createStatement();
+    	
+    	if(statement.execute("SELECT * FROM song")) {
+    		ResultSet rs = statement.getResultSet();
+    		while(rs.next()) {
+    			// put songs from library into the jTable somehow
+    		}
+    	}
+    }
+
+    
+    public void addSong(String fn) throws SQLException {
```

Comment:
These methods should take in a POJO object. We can create `Song` class and use it here. The caller of this method would create a new song object and pass it to this method.

For now we can just leave this as-is and I can fix this in future PRs.